core/server/services/url/UrlService.js

```diff
@@ -215,17 +223,23 @@ class UrlService {
             .getValue(options);
     }
 
-    reset() {
+    init() {
+        this.resources.fetchResources();
+    }
+
+    reset(removeListeners = true) {
```

Comment:
Let's use the options pattern? We use it everywhere. This avoid having functions with x arguments and you need to pass null or default values.

> reset(options = {removeListeners: true}) {

Reply:
Sure :sunglasses: